tests/test_spherical_expansions.py

```diff
@@ -40,22 +39,22 @@ def test_vector_expansion_coeffs(self):
         # Default types are float32 so we cannot get higher accuracy than 1e-7.
         # Because the reference value have been cacluated using float32 and
         # now we using float64 computation the accuracy had to be decreased again
-        assert equistore.operations.allclose(tm_ref, tm, atol=1e-5, rtol=1e-5)
+        assert metatensor.torch.allclose(tm_ref, tm, atol=1e-5, rtol=1e-5)
 
     def test_spherical_expansion_coeffs(self):
-        tm_ref = equistore.core.io.load_custom_array("tests/data/spherical_expansion_coeffs-ethanol1_0-data.npz", equistore.core.io.create_torch_array)
+        tm_ref = metatensor.torch.load("tests/data/spherical_expansion_coeffs-ethanol1_0-data.npz")
         spherical_expansion_calculator = SphericalExpansion(self.hypers, self.all_species, device=self.device)
         with torch.no_grad():
             tm = spherical_expansion_calculator.forward(**self.batch)
         # Default types are float32 so we cannot get higher accuracy than 1e-7.
         # Because the reference value have been cacluated using float32 and
         # now we using float64 computation the accuracy had to be decreased again
-        assert equistore.operations.allclose(tm_ref, tm, atol=1e-5, rtol=1e-5)
+        assert metatensor.torch.allclose(tm_ref, tm, atol=1e-5, rtol=1e-5)
 
     def test_spherical_expansion_coeffs_alchemical(self):
         with open("tests/data/expansion_coeffs-ethanol1_0-alchemical-hypers.json", "r") as f:
             hypers = json.load(f)
-        tm_ref = equistore.core.io.load_custom_array("tests/data/spherical_expansion_coeffs-ethanol1_0-alchemical-seed0-data.npz", equistore.core.io.create_torch_array)
+        tm_ref = metatensor.torch.load("tests/data/spherical_expansion_coeffs-ethanol1_0-alchemical-seed0-data.npz")
         torch.manual_seed(0)
         spherical_expansion_calculator = SphericalExpansion(hypers, self.all_species, device=self.device)
```

Comment:
Should be add the scripting test here? 

```suggestion
        spherical_expansion_calculator = SphericalExpansion(hypers, self.all_species, device=self.device)
        spherical_expansion_calculator = torch.jit.script(spherical_expansion_calculator)
```